writing_checks/README.md

```diff
@@ -468,6 +468,38 @@ whereas if it run on a system with ALPS it will be translated to
 ddt --offline aprun ...
 ```
 
+## Custom Launchers
+
+ReFrame permits the simple addition of custom scheduler launchers. A launcher that invokes `mpirun` can for example be added as follows:
```

Comment:
Yes, that might be unclear to some readers. 
The first occurrence of the word 'launcher' is in the subsection `## Executable Options`, so one might could add it there. However, as it is a frequent word in the whole section `# Customising the Run of a Test`, such that I think it could be better to add it right there (after `ReFrame offers several other options for customizing the behavior of regression tests.` and with a proper transition to it).
What do you think?